firefox-ios/Client/AccessoryViewProvider.swift

```diff
@@ -163,54 +141,42 @@ class AccessoryViewProvider: UIView, Themeable {
     }
 
     private func setupLayout() {
-        translatesAutoresizingMaskIntoConstraints = false
         setupSpacer(leadingFixedSpacer, width: UX.fixedLeadingSpacerWidth)
         setupSpacer(trailingFixedSpacer, width: UX.fixedTrailingSpacerWidth)
 
-        if showCreditCard {
-            let cardStackViewForBarButton = UIBarButtonItem(customView: cardButtonStackView)
-            cardStackViewForBarButton.accessibilityTraits = .button
-            cardStackViewForBarButton.accessibilityLabel = .CreditCard.Settings.UseSavedCardFromKeyboard
```

Comment:
It seems this was originally an accessibility label, but with the update it's a label text now in a stack view. Why was this changed? curious to understand this more in case I'm missing something!